suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -18,81 +18,153 @@ import {
+        const formatTransactionDate = () => {
+            const { blockHeight, blockTime } = transaction;
+            if (!blockTime || blockHeight === 0) return null;
+            // TODO this is just MVP and should be properly formatted in next PR
+            return new Date(blockTime * 1000).toLocaleString('en-US', {
+                month: 'short',
+                day: 'numeric',
+                hour: 'numeric',
+                minute: 'numeric',
+            });
+        };
 
-    return (
-        <TouchableOpacity
-            onPress={() => handleNavigateToTransactionDetail()}
-            style={applyStyle(transactionListItemStyle)}
-        >
-            <Box flexDirection="row" alignItems="center">
-                <Box style={applyStyle(iconStyle)}>
-                    <Icon name={transactionIconMap[transaction.type] ?? 'placeholder'} />
-                </Box>
+        const handleNavigateToTransactionDetail = () => {
+            navigation.navigate(RootStackRoutes.TransactionDetail, {
+                txid: transaction.txid,
+            });
+        };
+
+        const transactionTypeProperties = transactionTypePropertiesMap[transaction.type];
+
+        return (
+            <TouchableOpacity
+                onPress={() => handleNavigateToTransactionDetail()}
+                style={applyStyle(transactionListItemStyle, { isFirst, isLast })}
+            >
+                <TransactionListItemIcon
+                    cryptoIconName={transaction.symbol}
+                    transactionType={transaction.type}
+                />
                 <Box>
-                    <Text>{transactionTextMap[transaction.type] ?? transaction.type}</Text>
-                    <Text>{getTransactionTimestamp()?.toLocaleTimeString()}</Text>
+                    <Box style={applyStyle(titleCointainerStyle)}>
+                        <Text>{transactionTypeProperties.prefix}</Text>
+                        <Box style={applyStyle(addressStyle)}>
+                            <Text
+                                variant="label"
+                                color="gray600"
+                                numberOfLines={1}
+                                ellipsizeMode="middle"
+                            >
+                                {transaction.txid}
+                            </Text>
+                        </Box>
+                    </Box>
+                    <Text variant="hint" color="gray600">
+                        {formatTransactionDate()}
+                    </Text>
+                </Box>
+
+                <Box alignItems="flex-end">
+                    <Box flexDirection="row">
+                        <Text color={transactionTypeProperties.signColor}>
+                            {transactionTypeProperties.sign}{' '}
+                        </Text>
+                        <DiscreetText>{FiatAmountFormatter.format(fiatAmount ?? 0)}</DiscreetText>
```

Comment:
Please prefer usage of component formatters `<FiatAmountFormatter value={fiatAmount ?? 0) />`

Reply:
I believe @juriczech is changing these lines in PR https://github.com/trezor/trezor-suite/pull/7401.

Reply:
Good point, let's change them in PR for formatters.